lms/envs/common.py

```diff
@@ -2397,7 +2397,10 @@ def _make_locale_paths(settings):
     # User Leads app
     'openedx.features.user_leads',
 
-    'channels'
+    'channels',
+
+    'openedx.features.marketplace',
```

Comment:
Let's please move these under `# Philu Features` comment at line 2368 in the same file.